cli/viam/main.go

```diff
@@ -803,6 +803,38 @@ func main() {
 					},
 				},
 			},
+			{
+				Name:  "module",
+				Usage: "manage your hosted modules",
```

Comment:
```suggestion
				Usage: "manage your modules in Viam's registry",
```
Or we can ask fahmina what she thinks.

Reply:
Yours is better. Happy to go with that